backend/src/main/java/ddangkong/controller/exception/GlobalExceptionHandler.java

```diff
@@ -58,12 +68,12 @@ public ErrorResponse handleNoResourceFoundException(NoResourceFoundException e)
         return new ErrorResponse(ClientErrorCode.NO_RESOURCE_FOUND);
     }
 
-    @ExceptionHandler
+    @ExceptionHandler(value = {HttpRequestMethodNotSupportedException.class, HttpMediaTypeNotSupportedException.class})
```

Comment:
```suggestion
@ExceptionHandler(value = {
        HttpRequestMethodNotSupportedException.class, 
        HttpMediaTypeNotSupportedException.class
    })
```
사소한 의견) 요렇게 잘라줘도 가독성 좋을 것 같네요~!